common/scripted_triggers/wc_offmap_scripted_triggers.txt

```diff
@@ -48,15 +48,40 @@ yshaarj_offmap_decisions_enabled_trigger = {
 	has_character_modifier = follower_of_yshaarj
```

Comment:
So, I did 10 years test run from 605 to 615, the qiraji AI wasn't able to wake up C'Thun so they stayed locked down.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79078834-447fbd80-7d1c-11ea-8aa4-b99f19d18627.png)

</details>

Reply:
Well that's because they never freed C'Thun in lore this will be implemented by cho'gall freeing him after i merge cho'galls journey

Reply:
> Well that's because they never freed C'Thun in lore this will be implemented by cho'gall freeing him after i merge cho'galls journey

So, why you request my review here instead of Cho'gall's journey?

Reply:
cause if there are some big bugs here we have more time to fix them

Reply:
> cause if there are some big bugs here we have more time to fix them

The thing is... to fix these bugs you need to merge Cho'gall's journey, so this branch isn't playable and testable by itself.
So I would mark it WIP and wait for other branches to be merged instead of taking reviews from them.

Reply:
Bang. @sejtalk, notice me!

Reply:
There is another branch that makes Cho'gall go to ahn'qiraj or ulduar but it requires this one to be merged

Reply:
> There is another branch that makes Cho'gall go to ahn'qiraj or ulduar but it requires this one to be merged

Well, that's a main part of the branch. IIRC, the AI doesn't wake up the Old Gods thus they're not active unless the player play them.

Reply:
They do wake them by normal means but i can move the events to this branch so they can be awoken by chogall/chosen

Reply:
> They do wake them by normal means but i can move the events to this branch so they can be awoken by chogall/chosen

Link them, I'll take a look.

Reply:
I'll need to migrate them to new branch first but we need code review here too